modules/dxp/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/expression/FilterByCountIndividualSegmentsExpressionVisitorImpl.java

```diff
@@ -112,6 +114,61 @@ else if (Objects.equals("day", value)) {
 			"Unsupported operation with value " + value);
 	}
 
+	@Override
+	public FilterByCount.Day visitFunctionCallExpression(
+		@NotNull
+			IndividualSegmentsExpressionParser.FunctionCallExpressionContext
+				functionCallExpressionContext) {
+
+		ParseTree parseTree1 = functionCallExpressionContext.getChild(0);
+
+		Object functionName = parseTree1.accept(this);
+
+		if (Objects.equals("between", functionName)) {
```

Comment:
Maybe it is better to use Function.BETWEEN.toString() to compare. What do you think?

Reply:
Maybe it is better if I add another value to the enum with function in lowercase because we are requiring it in some lines.